pkg/scheduler/plugins/deviceshare/plugin.go

```diff
@@ -655,9 +659,16 @@ func New(obj runtime.Object, handle framework.Handle) (framework.Plugin, error)
 	registerPodEventHandler(deviceCache, handle.SharedInformerFactory(), extendedHandle.KoordinatorSharedInformerFactory())
 	go deviceCache.gcNodeDevice(context.TODO(), handle.SharedInformerFactory(), defaultGCPeriod)
 
+	gpuSharedResourceTemplatesCache := newGPUSharedResourceTemplatesCache()
+	registerGPUSharedResourceTemplatesConfigMapEventHandler(gpuSharedResourceTemplatesCache,
+		args.GPUSharedResourceTemplatesConfig.ConfigMapNamespace, args.GPUSharedResourceTemplatesConfig.ConfigMapName,
+		handle.SharedInformerFactory())
+
 	return &Plugin{
-		handle:                             extendedHandle,
-		nodeDeviceCache:                    deviceCache,
+		handle:                          extendedHandle,
+		nodeDeviceCache:                 deviceCache,
+		gpuSharedResourceTemplatesCache: gpuSharedResourceTemplatesCache,
+		gpuSharedResourceTemplatesMatchedResources: args.GPUSharedResourceTemplatesConfig.MatchedResources,
```

Comment:
If GPUSharedResourceTemplatesConfig is not set, the scheduler panics. This may be the reason why e2e keeps failing.

Reply:
![image](https://github.com/user-attachments/assets/928c5a77-2bf9-4771-9944-ca0fce638e61)

You need to modify the file config/rbac/scheduler_role.yaml so that the scheduler in the e2e environment can perceive the configmapv

Reply:
RBAC issue, solved.